File: PRAKTIKUM MODUL 1 C DAN PY/BAHASA C/PRAK104-2610817210007-MuhammadZakiMubarak.c
#include <stdio.h>

int main() {
    int shoe_a = 400000;
    int shoe_b = 350000;
    int discount_a = shoe_a - (13 * shoe_a / 100);
    int discount_b = shoe_b - (21 * shoe_b / 100);

    printf("Harga sepatu A adalah %d\n", shoe_a);
    printf("Harga sepatu B adalah %d\n", shoe_b);
    printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi %d\n", discount_a);
    printf("Sepatu B mendapat diskon 21%% sehingga harganya menjadi %d\n", discount_b);

    return 0;
}

File: PRAKTIKUM MODUL 1 C DAN PY/PYTHON/PRAK104-2610817210007-MuhammadZakiMubarak..py
shoe_a = 400000
shoe_b = 350000


discount_a = 0.13
discount_b = 0.21


(shoe_a * discount_a)
(shoe_b * discount_b)

print("Harga sepatu A adalah", shoe_a)
print("Harga sepatu A adalah", shoe_b)
print(f"Sepatu A mendapat diskon 13% sehingga harganya menjadi {int(shoe_a - (shoe_a * discount_a))}")
print(f"Sepatu B mendapat diskon 21% sehingga harganya menjadi {int(shoe_b - (shoe_b * discount_b))}")